protinfo_linux.go

```diff
@@ -19,38 +20,38 @@ func LinkGetProtinfo(link Link) (Protinfo, error) {
 		return pi, err
 	}
 
-	for _, m := range msgs {
-		ans := nl.DeserializeIfInfomsg(m)
+	for i := range msgs {
```

Comment:
Just to speed it up a littel bit and avoid copies. When you have for example few hundrest interfaces, or 300k routes, it speed up a littel bit, and it was really small change.